TaskList/TaskList/TaskRow.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  TaskRow.swift
+//  Task List
+//
+//  Created by Nick Ryan on 4/21/24.
+//
+
+import SwiftUI
+
+struct TaskRow: View {
+  @ObservedObject var taskStore = TaskStore()
+  let task: Task
+  
+  var body: some View {
+    ZStack(alignment: .topLeading) {
+      HStack() {
+        Text(task.title)
+          .font(.title2)
+          .fontWeight(.semibold)
+          .multilineTextAlignment(.leading)
+          .frame(width: nil, height: 35.0)
+        Spacer()
+        Image(systemName: task.isCompleted ? "checkmark.square" : "square")
```

Comment:
Nice! You should always strive to use the ternary operator when you're changing parameters of a SwiftUI view.